.github/ISSUE_TEMPLATE/release_issue_template.md

```diff
@@ -0,0 +1,51 @@
+# Release tracking issue
+
+<!--
+Note to onduty:
+
+This issue is used to track a single release of AMP from the canary cut to
+the 1% push to production, plus any cherry picks along the way.  Note that
+because the current onduty person handles pushing the build the previous onduty
+created to production a single issue will generally be updated by at least
+two onduty people.
+
+Once you have filed this issue post a link to it in the #release Slack channel.
+
+As each step is completed, check it off so the community can see the state of
+the current release at a glance.
+
+In addition to using the checkboxes to track the state of the release, use
+comments on this issue to track anything else you think might be of interest
+to the community regarding this release (e.g. to alert the community to delays).
+
+To document cherry picks that interrupt the default release process, add
+additional checkboxes at the stage the cherry pick was made, e.g. if the
+cherry pick is made after the initial canary release was at 1%, add these lines
+beneath the initial 1% checkbox and then check them as they occur:
+  - [ ] Cherry pick release [<RELEASE_NUMBER>](https://github.com/ampproject/amphtml/releases/tag/<RELEASE_NUMBER>) pushed to Dev Channel
```

Comment:
I think these are making the dead link checker in our linter unhappy. We can surround them with backticks. 
> NOTE If the link(s) above are not meant to resolve to a real webpage surrounding them with backticks will exempt them from the link checker.

Reply:
Thanks for catching that!